x-pack/plugins/security_solution/public/management/components/endpoint_execute_action/execute_action_host_response.test.tsx

```diff
@@ -43,17 +43,39 @@ describe('When using the `ExecuteActionHostResponse` component', () => {
     };
   });
 
+  const outputSuffix = 'output';
+
+  it('should show shell info and shell code', async () => {
+    render();
+    expect(renderResult.getByTestId(`test-executeResponseOutput-context`)).toBeTruthy();
+    expect(renderResult.getByTestId(`test-executeResponseOutput-shell`)).toBeTruthy();
+    expect(renderResult.getByTestId(`test-executeResponseOutput-cwd`)).toBeTruthy();
```

Comment:
nit: instead of `toBeTruthy()`, it may be semantically better to use `toBeInTheDocument()`. In this case, you can also use `queryByX` instead of `getByX`, so if the queried element is not present, the query function won't fail, but the `toBeInTheDocument()` will.

Reply:
Oh I like that better. Will update it

Reply:
done 177a31bc5570b1608cd05964179414bd857797ac